### DISCUSSION TITLE
CSS Layout Cookbook, Sticky Footers

### DISCUSSION THREAD

**Discussion by juloelker** [mdn/mdn-community#584] [Content] [answered]
I have been learning CSS grid recently, so I decided to try the first recipe on [(url) https://developer.mozilla.org/en-US/docs/Web/CSS/Layout_cookbook/Sticky_footers].  I found I could only get the footer to stick to the bottom of the page if I used min-height: 100vh on the .wrapper class, rather than min-height: 100%.  

I believe this is because vh will adjust to the client's viewport instead of adjusting to the surrounding content on the page.  

Hey MDN developers, I would like to do a pull request for this.  How should I proceed? I'm new here.

**Comment by AnilSeervi** [comment, marked as answer]
> I believe this is because vh will adjust to the client's viewport instead of adjusting to the surrounding content on the page.

Yes. You are right.

The thing thats happening in that example iframe is that a `height: 100%` is being applied to the both `html` and `body` tag by default, which is the reason why the example is still displaying the right layout.
   Here the `height : 100%` is basically imitating 100vh height on the `html`

**Comment by pepelsbey** [comment]
Closing as answered/settled.
